MANIFEST.in

```diff
@@ -0,0 +1,2 @@
+include versioneer.py
+include pydra_ml/_version.py
```

Comment:
This should get picked up automatically, since it's inside the package. (I know it's added automatically by versioneer.)

Reply:
any thoughts to this error then: https://github.com/nipype/pydra-ml/runs/3311997061?check_suite_focus=true

Reply:
Oh, no I agree we need `versioneer.py`. I was just noting that `pydra_ml/_version.py` should be found without being here. Apologies, it was meant as idle pedantry, not even really a suggestion that you do something different.

Reply:
ah ok. let me remove just that line and then merge.

on the dandi side one of our team members john created a new versioning package (`versioningit`). we may move this and other tools that currently use versioneer to it.